### PR TITLE
Onl 4036 card component

### PR DESCRIPTION
**Pull request by jaimeperezpedra**
Create a card style component.

Create the POC of trade-finance cards.

Design: [link](https://projects.invisionapp.com/share/HPQ2TPOQUM2#/screens/322819731)

Specs:
image

### CODE REVIEW

File: src/scss/components/ec-card/ec-card.story.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { storiesOf } from '@storybook/vue';
+import EcIcon from '@/components/ec-icon';
+
+const stories = storiesOf('card', module);
```
Comment (mcibique) [inline src/scss/components/ec-card/ec-card.story.js]: storiesOf(`'Card'`, module)

File: src/scss/components/ec-card/ec-card.story.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <div style="display: flex; height: 100vh; margin: 10px">
+      <div style="margin: auto; width: 33vw;">
+        <div class="ec-card">
+          Basic example card
+        </div>
+      </div>
+    </div>
+      `,
+  }))
+  .add('Trade finance cards', () => ({
```
Comment (mcibique) [inline src/scss/components/ec-card/ec-card.story.js]: We name the stories in lowercase.

File: src/scss/components/ec-card/_ec-card.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.ec-card {
+  @include box-shadow-level-1;
+  @include shape-border-radius;
+
+  padding: 24px;
```
Comment (mcibique) [inline src/scss/components/ec-card/_ec-card.scss]: The spec linked in PR says it should be 32px.

File: src/scss/components/ec-card/ec-card.story.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    components: { EcIcon },
+    template: `
+    <div style="display: flex; height: 100vh; margin: 10px">
+      <div style="margin: 10px; width: 66vw;">
+        <div class="ec-card">
+          <div class="ec-p--8">
+            <div style="font-size:25px; line-height: 35px;" class="ec-mb--24">
+              Credit line: EUR 1,000,0000.0
```
Comment (mcibique) [inline src/scss/components/ec-card/ec-card.story.js]: The dot is on wrong place, it should be `1,000,000.00`.

File: src/scss/components/ec-card/_ec-card.scss (hunk 3 above)
Comment (jaimeperezpedra) [reply src/scss/components/ec-card/_ec-card.scss]: Check it one more time, the image says in the card with the SVG that it have 32px of padding, but in the next, it says that now is 24px. I thought maybe the card with SVG have extra padding.

Let me ask Uve to confirm

Comment (jaimeperezpedra) [reply src/scss/components/ec-card/_ec-card.scss]: Uve confirm that, that should be with 24px and the CL 8px more

**Pull request merged**